tests/dashboard/dashboard-onboarding-admin.spec.js

```diff
@@ -55,7 +57,7 @@ test.describe('Onboarding Admin page tests', () => {
     await expect(table.locator('tbody')).toBeVisible()
     const rows = await table.locator('tbody').locator('tr').all()
 
-    expect(rows.length).toEqual(2);
+    expect(rows.length).toEqual(PORTAL_USER_ACCOUNT_SETUP_STEPS);
```

Comment:
Should be changed to this
```suggestion
    expect(rows.length).toEqual(portalUserAccountSteps.length);
```